### ISSUE TITLE
bad_json error with b2

### ISSUE THREAD

**Issue by marrie** [rclone/rclone#965]
I believe I filed this before. But I just can't remember.

[log link](https://www.dropbox.com/s/5gu8issmgwqmdev/outputsa.logaa.zip?dl=1)

rclone v1.33-79-g77b975dβ
Remote is b2.

Basically what's happening is I'm getting a complaint from the log that the file length is way too long?  The line you are looking for is this.

2016/12/19 10:45:40 Failed to sync: File name segment must be no more than 250 bytes (400 bad_json)

This looks very familiar to what I filed a few months ago. By the way the file back up just stops with no errors except with that one. Did the JSON  API Brake? I can back up everything else successfully as far as i can tell.

Thanks.When filing an issue, please include the following information if possible as well as a description of the problem.  Make sure you test with the [latest beta of rclone](http://rclone.org/downloads/).

If you've just got a question then please use the [rclone forum](https://forum.rclone.org/) instead of filing an issue.

> What is your rclone version (eg output from `rclone -V`)

> Which OS you are using and how many bits (eg Windows 7, 64 bit)

> Which cloud storage system are you using? (eg Google Drive)

> The command you were trying to run (eg `rclone copy /tmp remote:tmp`)

> A log from the command with the `-v` flag (eg output from `rclone -v copy /tmp remote:tmp`)

**Comment by ncw** [comment]
I'm not 100% sure what is going on from that log.

Can you run it without the `-q` flag and with the `-v` flag and post another log please?

**Comment by dsjr2006** [comment]
You did file this before. Please follow issue reporting procedure: https://github.com/ncw/rclone/blob/master/CONTRIBUTING.md#reporting-a-bug

The B2 API is responding that a filename is too long, but without verbose log output it is impossible to tell what that filename is.

**Comment by marrie** [comment]
Ok, here is the log again, uploaded to my GDrive



outputsa.logaa.zip <https://drive.google.com/uc?export=download&id=0BzuhYKTnNu0ka25sNzRwSHhJejg>
(8 KB)

**Comment by marrie** [comment]
I managed to recreate the log file and the issue with the verbose setting. It keeps getting stuck at the ~/library which I excluded. I tried doing a rclone cleanup but that didn't seem to work. I don't know if this log will help, but thought I'd attach it as I happen to look. Interesting that this should only effect my home dir which I'm backing up.

[output log file here.](https://github.com/ncw/rclone/files/679876/outputsa.logaa.zip)

Thanks.
